unit-tests/src/test/scala/java/lang/ProcessSuite.scala

```diff
@@ -9,18 +9,47 @@ import scala.scalanative.posix.{fcntl, unistd}
 import scala.io.Source
 
 object ProcessSuite extends tests.Suite {
+
+  private def blockTestOnConditions(process: Process): Unit = {
```

Comment:
The name of this method should start with `assert`. For example: `assertTerminates` or `assertExits`.

Reply:
I will change the name as recommended.  Probably to assertProcessExitOrTimeout().
Thank you for the review.

Reply:
Done. If the new name is not suitable, I can change it again.